packages/trace-viewer/src/sw/traceModelBackends.ts

```diff
@@ -129,14 +133,29 @@ export class FetchTraceModelBackend implements TraceModelBackend {
     const fileName = entries.get(entryName);
     if (!fileName)
       return;
-    return fetch('/trace/file?path=' + encodeURIComponent(fileName));
+    return this._server.readFile(fileName);
   }
 }
 
-function formatUrl(trace: string) {
-  let url = trace.startsWith('http') || trace.startsWith('blob') ? trace : `file?path=${encodeURIComponent(trace)}`;
+function formatUrl(trace: string, server: TraceViewerServerBackend) {
```

Comment:
Should this one move to `TraceViewerServerBackend` instead?

Reply:
No, because `TraceViewerServerBackend` is about loading files from disk, and most of this function is not.